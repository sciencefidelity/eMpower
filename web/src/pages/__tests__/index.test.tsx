/**
 * @jest-environment jsdom
 */

import React from "react"
import renderer from "react-test-renderer"
// import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

import IndexPage from "../index"

describe("IndexPage", () => {
  beforeAll(() => {
    const useStaticQuery = jest.fn();
    useStaticQuery.mockReturnValue({
      site: {
        siteMetadata: {
          title: `eMpower`
        }
      }
    })
  })

  const mockData = {
    site: {
      siteMetadata: {
        title: "eMpower"
      }
    }
  }

  it("renders correctly", () => {
    const tree = renderer
      .create(<IndexPage children={mockData} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it.todo("renders the site title")
})
