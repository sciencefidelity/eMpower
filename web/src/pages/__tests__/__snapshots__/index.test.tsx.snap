// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<div>
  <main />
</div>
`;
